.github/workflows/studio-release-tests.yml

```diff
@@ -19,10 +19,20 @@ jobs:
           pip install -r requirements.txt
           pip install -r requirements/requirements.test.txt
           pip install -e .
+          pip install coverage coveralls
       - name: Ensure browser is installed
         run: python -m playwright install --with-deps chromium
       - name: Run tests
         run: |
-          gunicorn --worker-class eventlet  -w 1 g2p.app:APP --no-sendfile --bind 0.0.0.0:5000 --daemon
+          coverage run --include g2p/app.py --parallel-mode \
+              -m gunicorn --worker-class eventlet  -w 1 g2p.app:APP --no-sendfile --bind 0.0.0.0:5000 --daemon
           sleep 5
-          cd g2p/tests && python test_studio.py
+          python g2p/tests/test_studio.py
+          pkill coverage
+          sleep 5
+          coverage combine
+          coverage xml
+      - uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          fail_ci_if_error: false # too many upload errors to keep "true"
```

Comment:
Thanks! I'm actually not certain that they are necessary, since codecov should combine the runs regardless, but we really aren't testing the app in the other tests.